axelrod/tests/test_appeaser.py

```diff
@@ -0,0 +1,23 @@
+"""
+Test for the appeaser strategy
+"""
+import unittest
+import axelrod
+
+class TestAppeaser(unittest.TestCase):
+
+    def test_strategy(self):
+        P1 = axelrod.Appeaser()
+        P2 = axelrod.Player()
+	P1.str = 'C';
+        self.assertEqual(P1.strategy(P2), 'C')
+	P1.history = ['C']
+	P1.history = ['C']
```

Comment:
I think this should be `P2.history`? Could you confirm and perhaps include another test to make sure this particular case (for which this worked) is also caught?